ci/generate-pipeline-steps.sh

```diff
@@ -112,36 +112,38 @@ if [ -z "${ENGINE_VERSION}" ]; then
     echo "Generating build steps for the first ${MAXIMUM_ENGINE_VERSION_COUNT_LOCAL} engine versions listed in unreal-engine.version"
     
     IFS=$'\n'
-    VERSIONS=$(cat < ci/unreal-engine.version)
+    VERSIONS=$(cat < ci/unreal-engine.version | tr -d '\r')
 
     # Turn on Firebase test steps
     echo "--- handle-firebase-steps"
     if [[ -n "${FIREBASE_TEST:-}" ]]; then        
         echo "--- insert-firebase-test-steps"
         COUNT=1
         for VERSION in ${VERSIONS}; do
-            echo --- handle-firebase-:${VERSION}-COUNT:${COUNT}
+            TRIMED_VERSION=$(sed 's/ *$//g' <<< ${VERSION})
+            echo --- handle-firebase-:${TRIMED_VERSION}-COUNT:${COUNT}
             if ((COUNT > MAXIMUM_ENGINE_VERSION_COUNT_LOCAL)); then
                 break
             fi
 
-            insert_firebase_test_steps "${VERSION}"
+            insert_firebase_test_steps "${TRIMED_VERSION}"
```

Comment:
```suggestion
            insert_firebase_test_steps "${TRIMMED_VERSION}"
```
should be TRIMMED instead of TRIMED everywhere :)